### PR TITLE
gitaction runner script enhancements

### PR DESCRIPTION
**Pull request by Polber**

### CODE REVIEW

**Review by damccorm — COMMENTED** [review]
Could you add some quick information on how this is run/deployed in a new markdown file in https://github.com/GoogleCloudPlatform/DataflowTemplates/tree/main/contributor-docs ? Long term it would be good to automate this as an action itself (using GitHub hosted runners)

**Review by Abacn — APPROVED** [review]

**Comment by Polber** [comment]
@damccorm Added an entry for gitactions and provisioning runners in the contribution guide.

File: contributor-docs/maintainers-guide.md [hunk 1]
```diff
@@ -56,4 +56,72 @@ This repo's code currently is mirrored in Google's internal source control syste
 6) Approve the change internally. At this point, the change will get automatically merged internally and externally.
 
 We are actively working to deprecate this process and use GitHub as the only source of truth.
-If you encounter unresolvable issues with this flow, please reach out to the Dataflow team directly.
\ No newline at end of file
+If you encounter unresolvable issues with this flow, please reach out to the Dataflow team directly.
+
+## Gitactions
```
Comment (damccorm) [inline contributor-docs/maintainers-guide.md:61]:
```suggestion
## GitHub Actions
```

File: contributor-docs/maintainers-guide.md [hunk 2]
```diff
@@ -56,4 +56,72 @@ This repo's code currently is mirrored in Google's internal source control syste
...
+
+There are several workflows that leverage gitactions to keep the repo healthy. Of these workflows, there are currently 2
```
Comment (damccorm) [inline contributor-docs/maintainers-guide.md:63]:
```suggestion
There are several workflows that leverage GitHub Actions to keep the repo healthy. Of these workflows, there are currently 2
```

**Review by damccorm — APPROVED** [review]
LGTM outside of 2 nits

File: contributor-docs/maintainers-guide.md (hunk 1 above)
Comment (Polber) [reply contributor-docs/maintainers-guide.md]: done.

File: contributor-docs/maintainers-guide.md (hunk 2 above)
Comment (Polber) [reply contributor-docs/maintainers-guide.md]: done.

**Pull request merged**